DFC.ServiceTaxonomy.Editor.sln

```diff
@@ -39,11 +39,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GetJobProfiles", "GetJobPro
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DFC.ServiceTaxonomy.GraphVisualiser", "DFC.ServiceTaxonomy.GraphVisualiser\DFC.ServiceTaxonomy.GraphVisualiser.csproj", "{A78E7B74-3AD3-493A-98B0-D23DD683C126}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DFC.ServiceTaxonomy.CustomFields", "DFC.ServiceTaxonomy.CustomFields\DFC.ServiceTaxonomy.CustomFields.csproj", "{2A5911E1-21EE-44D4-9BB6-92CB9994C79F}"
+EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DFC.ServiceTaxonomy.Cypher", "DFC.ServiceTaxonomy.Cypher\DFC.ServiceTaxonomy.Cypher.csproj", "{2E642150-CE42-4FA7-BED4-1FDC4337AC88}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DFC.ServiceTaxonomy.ContentPickerPreview", "DFC.ServiceTaxonomy.ContentPickerPreview\DFC.ServiceTaxonomy.ContentPickerPreview.csproj", "{0CDC10A8-D499-40B1-9B09-695F40C158BB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DFC.ServiceTaxonomy.ContentPickerPreview", "DFC.ServiceTaxonomy.ContentPickerPreview\DFC.ServiceTaxonomy.ContentPickerPreview.csproj", "{0CDC10A8-D499-40B1-9B09-695F40C158BB}"
```

Comment:
Project Guids have changed

Reply:
I think this is OK.
If we revert the GUID, I think VS will continually try and change it back, so we might as well just accept the new GUID.
https://github.com/dotnet/project-system/issues/1821